components/Home/CosmWasmDocumentationSection.vue

```diff
@@ -0,0 +1,42 @@
+<script lang="ts" setup>
+  import { Link } from '@/components/Ui';
+  import CosmWasmDocumentation from '@/assets/images/cosmwasm-documentation.png';
+</script>
+
+<template>
+  <div class="grid pb-16">
+    <p class="text-base text-gray-600 mb-4 dark:text-gray-900 leading-[150%]">Featured</p>
+    <Link
+      href="/developers/cosm_wasm/introduction"
+      class="grounded-radiants border-transparent rounded-2xl bg-white dark:bg-black-18 shadow-card dark:shadow-black"
+    >
+      <div class="flex">
+        <img class="w-[115px] mx-8 my-6" :src="CosmWasmDocumentation" />
+        <div class="mt-11">
+          <p class="text-gray-500 dark:text-gray-900">Dapp Developers</p>
+          <h2 class="heading-3 mt-1">Learn CosmWasm</h2>
+        </div>
+      </div>
+    </Link>
+  </div>
+</template>
+
+<style scoped>
+  .grounded-radiants {
```

Comment:
I think these styles from line 25 to 29 can be added as tailwind classes in line 11. If they are being overwritten, you can also add an exclamation sign before to make it !important, for example like this: `!relative`